portfolio/src/main/webapp/script.js

```diff
@@ -55,17 +55,13 @@ function rotateImages() {
 }
 
 function getCommentsUsingArrows() {
-  fetch('/data').then(response => response.json()).then((comments) => {
+  fetch('/data')
+    .then(response => response.json())
+    .then((comments) => {
     commentsList = document.getElementById('comment-container');
-    commentsList.innterHTML = '';
-    commentsList.appendChild(
-      createListElement(comments[0]));
-    commentsList.appendChild(
-      createListElement(comments[1]));
-    commentsList.appendChild(
-      createListElement(comments[2]));
-    commentsList.appendChild(
-      createListElement(comments[3]));
+    commentsList.innerHTML = '';
```

Comment:
Nit: +2 for the block inside the 2nd `.then`.

For reference: https://google.github.io/styleguide/jsguide.html#formatting-function-expressions